features/device_key/source/DeviceKey.cpp

```diff
@@ -283,7 +270,7 @@ int DeviceKey::generate_key_by_random(uint32_t *output, size_t size)
 
     mbedtls_entropy_free(entropy);
     delete entropy;
-
+    device_inject_root_of_trust(key_buff, actual_size);
```

Comment:
Error code needs to be recorded from here as well. The injection phase can fail as rell, so
```
ret = device_inject_root_of_trust(key_buff, actual_size);
```

Reply:
Changed